### DISCUSSION TITLE
Error fetching configuration information

### DISCUSSION THREAD

**Discussion by kenibarwick** [Blackymas/NSPanel_HA_Blueprint#745] [Q&A] [answered]
Hi, 

sorry if this is the incorrect place, however I am facing the above error when I try to install my NS Panel as a new device. 

![image](https://user-images.githubusercontent.com/3330132/235006314-631335c7-899e-447a-b62b-652c63f29112.png)

I did gain connection at first, but all I get is that error... 

What am I doing wrong please?

Thanks

Keni

**Comment by kenibarwick** [comment, marked as answer]
Ahh, it seems as though I have solved the issue. My Home Assistant instance wasn't connected to the server, even though the UI was working. 

Unless anyone objects I'll leave this here to help with any edge cases like me.

**Reply by edwardtfn** [reply]
> Unless anyone objects I'll leave this here to help with any edge cases like me.

I like the idea, but you can still close the discussion and other people will still be able to find it. It works even better when it is as an issue rather than a Discussion.
